src/app/technical-record/brakes/psv-brakes/psv-brakes.component.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { PsvBrakesComponent } from './psv-brakes.component';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { SharedModule } from '@app/shared/shared.module';
+import { TESTING_UTILS } from '../../../utils/testing.utils';
+
+describe('PsvBrakesComponent', () => {
+  let component: PsvBrakesComponent;
+  let fixture: ComponentFixture<PsvBrakesComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [SharedModule],
+      declarations: [PsvBrakesComponent],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA]
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(PsvBrakesComponent);
+    component = fixture.componentInstance;
+    component.brakes = TESTING_UTILS.mockPSVBrakes();
+    component.brakes.brakeForceWheelsNotLocked = TESTING_UTILS.mockBrakeForceWheelsNotLocked();
+    component.brakes.brakeForceWheelsUpToHalfLocked = TESTING_UTILS.mockBrakeForceWheelsUpToHalfLocked();
```

Comment:
TESTING_UTILS.mockPSVBrakes({
  brakeForceWheelsNotLocked: TESTING_UTILS.mockBrakeForceWheelsNotLocked(),
  brakeForceWheelsUpToHalfLocked : TESTING_UTILS.mockBrakeForceWheelsUpToHalfLocked()
})

`brakes` is the exposed variable